demos/build.gradle

```diff
@@ -17,7 +17,8 @@ subprojects {
   }
 
   dependencies {
-    compile 'javax.servlet:javax.servlet-api:3.1.0', project(':impl')
+    compile project(':impl')
```

Comment:
This should really not use the `java-library` plugin... because it's not a library.  I guess I could rework the main build.gradle to understand this or I could make this 'work' with the wrong plugin? Opinions?

Reply:
I don't mind using `java-library` there. Because having one sub-project not using it will just create a mess in the configuration for no particular gain. However, `compile` is deprecated from what I understood. So those should be `implementation`.